cmd/nerdctl/inspect.go

```diff
@@ -42,18 +42,34 @@ func newInspectCommand() *cobra.Command {
 	return inspectCommand
 }
 
+var validInspectType = map[string]bool{
+	"container": true,
+	"image":     true,
+}
+
 func addInspectFlags(cmd *cobra.Command) {
 	cmd.Flags().StringP("format", "f", "", "Format the output using the given Go template, e.g, '{{json .}}'")
 	cmd.RegisterFlagCompletionFunc("format", func(cmd *cobra.Command, args []string, toComplete string) ([]string, cobra.ShellCompDirective) {
 		return []string{"json"}, cobra.ShellCompDirectiveNoFileComp
 	})
+	cmd.Flags().String("type", "container", "Specify inspect-object type ('image' or 'container') (default 'container')")
```

Comment:
You don't need to add `(default 'container')` here. The default help text is automatically generated by cobra.

Reply:
Done